test/Chirp.Infrastructure.Tests/CheepRepositoryCreateTest.cs

```diff
@@ -45,19 +45,20 @@ public CheepRepositoryUnitTests()
     }
 
     [Fact]
-    public async Task UnitTestCreateMethod()
+    public async void UnitTestCreateMethod()
```

Comment:
Nice to know. Didn't knew, void could be returned only, and not wrapped inside a Task, when the method is async.
Thanks for giving me the oppoturnity to learn that :)